StoreApp/Models/Color.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace StoreApp.Models
+{
+    public class Color
```

Comment:
DB models are about DB structure? not about behavior
so it is usually no need to have constructors and methods
remove it please
in other models as well